code/modules/projectiles/guns/smgs.dm

```diff
@@ -440,3 +440,65 @@
     if(.)
         click_empty(user)
     return FALSE
+
+//-------------------------------------------------------
+//M1921 //Based on the Model 1921.
+
+/obj/item/weapon/gun/smg/m1921
+	name = "\improper M1921 Thompson submachinegun"
+	desc = "A modern day copy of a classic design, the Model 1921 Thompson submachine gun was known for its use in WW2 and by mobsters during the period which earned its nickname 'the Chicago Typewriter'"
```

Comment:
```suggestion
	desc = "A modern day copy of an ancient design, the Model 1921 Thompson submachine gun was known for its American use in WW2 and by high-profile mobsters during the prohibition era which earned its nickname 'the Chicago Typewriter'."
```
Alright, I am summoned, time to fix some typos.

also classic to ancient since in CM it's basically so so old by now.

Reply:
Changing American to Allied - it was used extensively by British and Canadian troops in the European theater. I didn't specify country use because of how many nations actually used it.